compose/README.md

```diff
@@ -1,100 +1,49 @@
+## Core
 
-## Download submodules after downloading the main project:
-```bash
-git submodule update --init --recursive
-```
-Set this property to always update submodules on git checkout/pull/reset:
-```bash
-git config --global submodule.recurse true
-```
+Compose Multiplatform development is going
+in [compose-multiplatform-core](https://github.com/JetBrains/compose-multiplatform-core) repository.
+There Compose Multiplatform team and contributors adopt Jetpack Compose for iOS, Desktop and Web targets.
 
-## General requirements
-- Java 11 (should be specified in JAVA_HOME)
-- [macOs/Linux] Android SDK downloaded via `./compose/frameworks/support/jbdeps/android-sdk/downloadAndroidSdk`
-- [Windows] Android SDK downloaded from Android Studio and specified in ANDROID_SDK_ROOT
-- [Optional, macOs] For AppCode IDE, specify environment variable `export ANDROID_SDK_ROOT=[YOUR_PATH]/compose-jb/compose/frameworks/support/jbdeps/android-sdk/darwin` (in ~/.zshrc)
+### [Get started](https://github.com/JetBrains/compose-multiplatform-core/blob/jb-main/MULTIPLATFORM.md)
 
-## Developing in IDE
-1. Download Android Studio from [the official site](https://developer.android.com/studio/archive) (it is mandatory to use the version, written [here](https://github.com/JetBrains/androidx/blob/jb-main/gradle/libs.versions.toml#L11)). As an alternative you can use IDEA, which is compatible with [this AGP version](https://github.com/JetBrains/androidx/blob/jb-main/gradle/libs.versions.toml#L5), or you can disable Android plugin in IDEA plugins, to develop non-Android targets.
-2. [macOs/Linux] Download Android SDK via `./compose/frameworks/support/jbdeps/android-sdk/downloadAndroidSdk`
-2. [Windows] Download Android SDK via [Android Studio](https://developer.android.com/studio/intro/update#sdk-manager) and specify it in ANDROID_SDK_ROOT environment variable. Components, their versions and folder structure should be the same as in downloaded via script `./compose/frameworks/support/jbdeps/android-sdk/downloadAndroidSdk` SDK for other platforms.
-3. Open `compose/frameworks/support` folder either in Android Studio or in IDEA or in AppCode with KMM plugin.
-4. Specify Gradle JVM to use JDK 11 in InteliJ IDEA Preferences (`Build, Execution, Deployment -> Build Tools -> Gradle`)
+## Skiko
 
-## Scripts
-Publish artifacts to the local directory `out/androidx/build/support_repo/org/jetbrains/compose`:
-```
-export COMPOSE_CUSTOM_VERSION=0.0.0-custom
-./scripts/publish
-```
-(on Windows it doesn't build at the moment, the issue is in the line `packageInspector(project, project(":compose:ui:ui-inspection")` in `compose/frameworks/support/compose/ui/ui/build.gradle`)
+Compose Multiplatform uses [Skiko](https://github.com/JetBrains/skiko). A low-level library that hides platform
+complexity and provides a simple interface for rendering, event handling, window management, and other features. Skiko
+uses Skia as graphical API.
```

Comment:
Should we point to our fork of skia?

Reply:
Not sure, from my point it should be handled in skiko